tests/setup_suite.bash

```diff
@@ -176,6 +176,10 @@ flox_cli_vars_setup() {
   unset FLOX_PROMPT_ENVIRONMENTS _FLOX_ACTIVE_ENVIRONMENTS
   export FLOX_DISABLE_METRICS='true'
   unset FLOX_FEATURES_USE_CATALOG
+  # Unset catalog feature flags
+  unset FLOX_FEATURES_USE_CATALOG
```

Comment:
If you don't unset these in the test suite you'll leak your shell's environment into the test suite and some tests will fail.